airbyte-integrations/connectors/source-monday/source_monday/extractor.py

```diff
@@ -89,6 +92,9 @@ def try_extract_records(self, response: requests.Response, field_path: List[Unio
             extracted = dpath.util.get(response_body, path, default=[])
 
         if extracted:
+            if isinstance(extracted, list) and None in extracted:
+                logger.warning(f"Record with null value received; errors: {response_body.get('errors')}")
+                return [x for x in extracted if x]
```

Comment:
Let's make a `generator` instead, if possible

Reply:
Declarative CDK build upon simple functions, not the generators right now, need to rework CDK first.